src/transformers/file_utils.py

```diff
@@ -232,6 +232,7 @@
 TRANSFORMERS_CACHE = os.getenv("TRANSFORMERS_CACHE", PYTORCH_TRANSFORMERS_CACHE)
 
 WEIGHTS_NAME = "pytorch_model.bin"
+FLAX_WEIGHTS_NAME = "flax_model.msgpack"
```

Comment:
from_pretrained and save_pretrained didn't work previously with flax really. Everything was done in PyTorch. Is this weight name ok?

Reply:
Works for me!